storage/container_storage_interface/persistent-storage-csi-ovirt.adoc

```diff
@@ -0,0 +1,27 @@
+[id="persistent-storage-csi-ovirt"]
+= Red Hat Virtualization (oVirt) CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-ovirt
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Red Hat Virtualization (oVirt).
+
+Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
+provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
+
+To create CSI-provisioned PVs that mount to Red Hat Virtualization (oVirt) storage assets, {product-title} installs the oVirt CSI Driver Operator and the oVirt CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* The **oVirt CSI Driver Operator** provides a default StorageClass that you can use to create PVCs.
+
+* The **oVirt CSI driver** is a custom resource (CR) that enables you to create and mount oVirt PVs.
+
+== About CSI
+Storage vendors have traditionally provided storage drivers as part of Kubernetes. With the implementation of the Container Storage Interface (CSI), third-party providers can instead deliver storage plug-ins using a standard interface without ever having to change the core Kubernetes code.
+
+CSI Operators give {product-title} users storage options, such as volume snapshots, that are not possible with in-tree volume plug-ins.
+
+.Additional resources
+* xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Configuring CSI volumes]
```

Comment:
@rolfedh oVirt Operator is added to OCP 4.6 by default (see https://issues.redhat.com/browse/STOR-380). Can you recommend a SME who could review this assembly's content for technical accuracy? Thanks!